src/homepage/howTo/custom-listeners.html

```diff
@@ -58,12 +58,16 @@ <h2><b>What is Mailbox Listener</b></h2>
                 </p>
                 <ul>
                     <li>
-                        <b>Group Registration</b>: If a Listener is registered as the Group Registration, it will receive all of events fired by James Mailbox System.
+                        <b>Group Registration</b>: a Group Registration will listen all events fired and delivery each event only once per group within a distributed topology.
+                        That means if there are three listeners register to the same group, each time an event is fired, only one in three listeners can receive that event.
```

Comment:
s/three listeners register/three listeners register**ed**
s/only one in three listeners/only one of these three listeners